File: comments/src/main/java/net/particify/arsnova/comments/model/serialization/StringToUuidConverter.java
package net.particify.arsnova.comments.model.serialization;

import java.util.UUID;
import org.springframework.core.convert.converter.Converter;
import org.springframework.stereotype.Component;

@Component
public class StringToUuidConverter implements Converter<String, UUID> {
  @Override
  public UUID convert(final String value) {
    return UuidHelper.stringToUuid(value);
  }
}

File: authz/src/main/kotlin/net/particify/arsnova/authz/model/serialization/StringToUuidConverter.kt
package net.particify.arsnova.authz.model.serialization

import org.springframework.core.convert.converter.Converter
import org.springframework.stereotype.Component
import java.util.UUID

@Component
class StringToUuidConverter : Converter<String, UUID?> {
  override fun convert(value: String): UUID? {
    return UuidHelper.stringToUuid(value)
  }
}
